translation.rst

```diff
@@ -292,6 +292,35 @@ To manage these situations, Symfony follows the `ICU MessageFormat`_ syntax by
 using PHP's :phpclass:`MessageFormatter` class. Read more about this in
 :doc:`/translation/message_format`.
 
+Translatable Objects
```

Comment:
you may need to add a versionadded directive block

also in the `reference/twig_reference.rst` file tho i do not know how to do it for an inline addition :s

Reply:
Added for this section. For `reference/twig_reference.rst` also unsure about the best approach there.

Reply:
Took a stab at adding a `versionadded` block in the twig reference.